scripts/checkpatch.pl

```diff
@@ -5508,11 +5508,30 @@ sub process {
 				}
 			}
 
+
+			# We need this for the custom labwc check below. It
+			# avoid false positives with do { } while (); etc.
+			my $starts_with_if_while_etc = 0;
+			if ($s =~ /^\+\s*(if|while|for|switch).*/) {
+				$starts_with_if_while_etc = 1;
+			}
+
 			# Find out what is on the end of the line after the
 			# conditional.
 			substr($s, 0, length($c), '');
 			$s =~ s/\n.*//g;
 			$s =~ s/$;//g;	# Remove any comments
+
+			# CUSTOM: labwc check to find if-condiationals without opening braces because
+			# (as opposed to Linux coding style) we use braces for single statement blocks
+			#
+			# include/ssd-internal.h contains a macro that we can't deal with, so ignore that
+			#
+			if ($starts_with_if_while_etc && !length($s)
+					&& $filename ne "include/ssd-internal.h") {
+				CHK("BRACES", "open brace { expected after if confidional");
```

Comment:
`conditional`, also in comment above

Also: maybe we should use if-conditionals without braces here. Just because :)

Reply:
Sorted